corona-warn-app/cwa-app-android
# DGC related Testresult-UI dynamics (EXPOSUREAPP-7427)

This PR addresses Exposureapp-7427.
BulletPoints in TestresultView should adjust content dynamically depending on PCR/RAT/Cert supported..
Please check the Overall-Ticket or Figma(best way) for all possible routes.

How-To-Test (examples):
- PCR Test with Example QR Code 
- Check the Pending testresult Screen

![image](https://user-images.githubusercontent.com/46747780/120326432-feebce80-c2e8-11eb-848a-de7b7c68c0e3.png)
(should contain bottom point)


- Scan RAT Test with Example QR Code
- Check the pending testresult Screen

![image](https://user-images.githubusercontent.com/46747780/120326629-38243e80-c2e9-11eb-873c-fc42d1fb275d.png)
(should contain bottom point)

Note: if the overall implementation is ready, and if the isAdvancedGiven attribute is set, "By your wish, no test cert ..." should be shown. (Consent will come with other PR/Subtask)
